app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -39,82 +39,95 @@ private const val KEY_REQUEST_TOKEN = "KEY_REQUEST_TOKEN"
 
 class HomeFragment :
     BaseMvpFragment<HomeContract.View, HomeContract.Presenter>(R.layout.fragment_home),
-    HomeContract.View,
-    OnHomeItemsClickListener {
+    HomeContract.View {
 
     companion object {
-        fun create() = HomeFragment()
+        fun create(): HomeFragment = HomeFragment()
     }
 
     override val presenter: HomeContract.Presenter by inject()
 
     private val binding: FragmentHomeBinding by viewBinding()
 
-    private val mainAdapter: TokenAdapter by lazy {
+    private val mainAdapter: TokenAdapter by unsafeLazy {
         TokenAdapter(this)
     }
 
     private val browseAnalytics: BrowseAnalytics by inject()
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        with(binding) {
-            val commonTitle = getString(R.string.app_name)
-            val beta = getString(R.string.common_beta)
-            val color = getColor(R.color.textIconSecondary)
-            titleTextView.text = SpanUtils.highlightText("$commonTitle $beta", beta, color)
-
-            mainRecyclerView.layoutManager = LinearLayoutManager(requireContext())
-            mainRecyclerView.adapter = mainAdapter
-
-            with(actionButtonsView) {
-                onBuyItemClickListener = {
-                    presenter.onBuyClicked()
-                }
-                onReceiveItemClickListener = {
-                    replaceFragment(ReceiveSolanaFragment.create(null))
-                }
-                onSendClickListener = {
-                    replaceFragment(SendFragment.create())
-                }
-                onSwapItemClickListener = {
-                    replaceFragment(OrcaSwapFragment.create())
-                }
-            }
+        binding.setupView()
```

Comment:
suggest to use name binding.bindView()

Reply:
it already binded by viewBinding() :) 
I can rename to initView